platform/mock-local-home-platform.ts

```diff
@@ -159,15 +146,4 @@ export class MockLocalHomePlatform implements MockUDPListener {
     this.localDeviceIds.set(device.id, device.verificationId);
     this.onNewDeviceIdRegistered(device.verificationId);
   }
-
-  public addUDPScanConfig(scanConfig: UDPScanConfig) {
-    this.udpScanConfigs.push(scanConfig);
```

Comment:
do we still need the `udpScanConfigs` field and the `ScanConfig` type?

Reply:
Removed.